base/resolver_node_lookup.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
The naming on these files is odd. I think it would be more "standard" to name this "node_lookup_resolver.go" and the other "socket_resolver.go". Those would comport with the actual struct names.